experimental/examples/prometheus/prometheus.yml

```diff
@@ -6,4 +6,4 @@ scrape_configs:
     # metrics_path defaults to '/metrics'
     # scheme defaults to 'http'.
     static_configs:
-    - targets: ['localhost:9464']
```

Comment:
That looks like it should still be `localhost` right? :slightly_smiling_face:

Reply:
Fixed!